src/App.js

```diff
@@ -1,16 +1,41 @@
 import React from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog.js';
+import { useState } from 'react';
+
 
 const App = () => {
+  const [chatMessageData, setChatMessages] = useState(chatMessages);
+
+  const updateChatMessages = updatedMessage => {
+    const chats = chatMessageData.map(message => {
+      if (message.id === updatedMessage.id) {
+        return updatedMessage;
+      } else {
+        return message;
+      }
+    });
+    setChatMessages(chats);
+  };
+
+
+  let likes = 0;
+  for (let message of chatMessageData ){
+    if (message.liked) {
+      likes++;
+    }
+  }
+
+
   return (
     <div id="App">
       <header>
-        <h1>Application title</h1>
+        <h1>Chat</h1>
+        <h3>Likes: {likes} ❤️s  </h3>
```

Comment:
Make sure you don't skip levels when using `<h1>-<h6>` tags for headings! Browsers use heading tags to construct a table of contents, so the different numbers on heading tags should be treated as different heading levels rather than as different style options. It's best practice to start headings at `h1` and then move to `h2` then `h3` and so on as you need to nest sections. If all headers are at the top level, then all header tags should be `h1`. CSS should be used to resize and change the style as needed.

You can read more about heading tag best practices here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#usage_notes